hikari/impl/interaction_server.py

```diff
@@ -186,13 +186,10 @@ def __init__(
         self._is_closing = False
         self._listeners: typing.Dict[typing.Type[base_interactions.PartialInteraction], typing.Any] = {}
         self._loads = loads
+        self._nacl = nacl
```

Comment:
Idk if I like this tbh. I am going to have a look at an idea I had and see if mypy would be happy about it and come back to this